packages/node-core/src/indexer/testing.service.ts

```diff
@@ -134,6 +135,8 @@ export abstract class TestingService<A, SA, B, DS> {
         await this.sequelize.createSchema(`"${schema}"`, {});
       }
```

Comment:
Is this necessary now?

Reply:
yeah, schema needs to be create before calling `initDbSchema`